libmscore/rest.cpp

```diff
@@ -582,36 +582,40 @@ int Rest::computeLineOffset(int lines)
             // of this segment (for measure rests, of the whole measure) in all opposite voices. 
             // Ignore stems and articulations, because which multi-voice they are at the opposite end.
             int upOffset = up ? 1 : 0;
-            int firstTrack = staffIdx() * 4;
             int line = up ? 10 : -10;
-            int extraOffsetForFewLines = lines < 5 ? 2 : 0;
-            bool isMeasureRest = durationType().type() == TDuration::DurationType::V_MEASURE;
-            Segment* seg = isMeasureRest ? measure()->first() : s;
-            while (seg) {
-                  for (const int& track : { firstTrack + upOffset, firstTrack + 2 + upOffset }) {
-                        Element* e = seg->element(track);
-                        if (e && e->isChord()) {
-                              Chord* chord = toChord(e);
-                              StaffGroup staffGroup = staff()->staffType(chord->tick())->group();
-                              for (Note* note : chord->notes()) {
-                                    int nline = staffGroup == StaffGroup::TAB
-                                          ? note->string() * 2
-                                          : note->line();
-                                    nline = nline - centerDiff;
-                                    if (up && nline <= line) {
-                                          line = nline - extraOffsetForFewLines; 
-                                          if (note->accidentalType() != AccidentalType::NONE)
-                                                line--;
-                                          }
-                                    else if (!up && nline >= line) {
-                                          line = nline + extraOffsetForFewLines;
-                                          if (note->accidentalType() != AccidentalType::NONE)
-                                                line++;
+
+            // For compatibility reasons apply automatic collision avoidance only if y-offset is unchanged 
+            if (qFuzzyIsNull(offset().y())) {
```

Comment:
I think instead of this you should be checking the STYLED flag on the OFFSET property.  That's way to do these sort of things where we need different behavior depending on whether the use has customized some property or not.

Reply:
Hmm, except I guess it's not actually a styled property to begin with.  Probably should have been all along - a way for the user to set the default offset in multivoice situations.

Since it isn't a styled property, instead you should probably be checking against the propertyDefault() rather than null, even though null does happen to be the default.  I don't know, not an important distinction I guess.